### PR TITLE
Added some pneumatic system framework and increased some cleanliness of code

### PR DESCRIPTION
**Pull request by Smwitte27**

### CODE REVIEW

File: src/main/java/frc/robot/commands/LiftRobot.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+
+/**
+ * An example command.  You can replace me with your own command.
+ */
+public class LiftRobot extends Command {
```
Comment (leskaa) [inline src/main/java/frc/robot/commands/LiftRobot.java:16]: What is this command supposed to do? Is it used to lift the robot up at the end for climbing onto the platform?

File: src/main/java/frc/robot/subsystems/HatchEjection.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * An example subsystem.  You can replace me with your own Subsystem.
+ */
+public class HatchEjection extends Subsystem {
```
Comment (leskaa) [inline src/main/java/frc/robot/subsystems/HatchEjection.java]: Try changing the name to HatchEjector so it is a noun to represent the physical subsystem.

File: src/main/java/frc/robot/commands/LiftRobot.java (hunk 1 above)
Comment (Smwitte27) [reply src/main/java/frc/robot/commands/LiftRobot.java:16]: That is what it is going to be, using pistons, I'm currently having 4 pistons allocated to this part

**Review by leskaa — APPROVED** [review]
Looks good. 👍

**Pull request merged**